pkg/splunk/enterprise/configuration.go

```diff
@@ -748,6 +736,53 @@ func updateSplunkPodTemplateWithConfig(client splcommon.ControllerClient, podTem
 	}
 }
 
+// getLivenessProbe the probe for checking the liveness of the Pod
+// uses script provided by enterprise container to check if pod is alive
+func getLivenessProbe(spec *enterprisev1.CommonSplunkSpec, additionalDelay int32) *corev1.Probe {
+	livenessDelay := livenessProbeDefaultDelay + additionalDelay
+	if spec.LivenessInitialDelaySeconds > livenessDelay {
+		livenessDelay = spec.LivenessInitialDelaySeconds
+	}
+
```

Comment:
probably a log msg here saying what is the value of livenessDelay. Might be helpful later while debugging issues related to this.